src/strings.js

```diff
@@ -1,25 +1,25 @@
 const sayHello = (string) => {
-  // your code here
+  return 'Hello, ' + string + '!';
```

Comment:
This is just an FYI - you can also use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) for this so:

```js
return `Hello, ${string}!`
```

It's great because using the `${}` syntax, you don't have to escape or concatenate strings.